docs/en/guides/developer/transactional.md

```diff
@@ -0,0 +1,50 @@
+# Transactional (ACID) support
+
+INSERTs into one partition in one table of the MergeTree\* family up to `max_insert_block_size` rows are transactional (ACID):
+
+### Atomic
+
+INSERTs succeed or are rejected as a whole- if a confirmation is sent to the client, then all rows were inserted; if an error is sent to the client, then no rows were inserted.
+
+### Consistent
+
+If an INSERT succeeded, then all rows were inserted and they do not violate table constraints; if constraints were violated, then no rows were inserted.
+
+### Isolated
+
+Clients observe a consistent snapshot of the table - either the state of the table before an INSERT, or the state after a successful INSERT; no partial state is seen.
+
+### Durable
+
+A successful INSERT is written to the filesystem before sending a confirmation to the client, on single replica or multiple replica systems (this is controlled by the `insert_quorum` setting), and ClickHouse can ask the operating system to sync the filesystem data on the storage media (this is controlled by the `fsync_after_insert` setting).
+
+### Details
+
+- If a table has multiple partitions and the INSERT covers multiple partitions - then insertion into each partition is transactional on its own.
+
+- INSERTs into multiple tables with one statement is possible if materialized views are used.
```

Comment:
Probably something like that is more correct? But I'm not a native speaker.
```suggestion
- Atomic INSERT into multiple tables with one statement is possible if materialized views are used.
```
@alexey-milovidov How do you think, do we need to cover here what happens if MV insert fails and how deduplication works for MV?

```suggestion
- INSERTs into multiple tables with one statement are possible if materialized views are used.
```

Reply:
Let's merge, then improve...